uint32 height
uint32 width
uint32 channels
uint32 buffer_count
uint32 frame_size
uint32 shm_num
uint32 init_id
